certified-connectors/QPP NG Integration Hub/apiDefinition.swagger.json

```diff
@@ -533,16 +537,17 @@
     "/api/v1/assetService/getAsset": {
       "get": {
         "summary": "Get Asset Meta Data",
-        "description": "Get Asset Data for given Asset Id.",
+        "description": "Get Asset Data for given Asset Id.For more information Check the github documentation.",
```

Comment:
Please leave space after period. "Id.For"

Reply:
Updated for all description for operations.